src/components/Link/types.ts

```diff
@@ -7,5 +7,5 @@ export enum LinkVariants {
   /** link is being rendered in a footer|header */
   snwFlyoutLink = TextVariants.snwFlyoutLink,
   /** standard link */
-  link = TextVariants.link,
+  titleCaseAndUnderline = TextVariants.titleCaseAndUnderline,
```

Comment:
If we're replacing `link` with `titleCaseAndUnderline` why change the name at all?  Why not just leave it `link` and change the style.  The `LinkVariants` names weren't intended to describe the styles but instead capture the usecase of how these links were used.